display_information/display-dimensions/src/main/java/com/esri/samples/display_dimensions/DisplayDimensionsSample.java

```diff
@@ -0,0 +1,147 @@
+import com.esri.arcgisruntime.mapping.view.MapView;
+import javafx.application.Application;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Alert;
+import javafx.scene.control.CheckBox;
+import javafx.scene.control.Label;
+import javafx.scene.layout.*;
+import javafx.scene.paint.Paint;
+import javafx.stage.Stage;
+
+import java.io.File;
+import java.util.Objects;
+
+public class DisplayDimensionsSample extends Application {
+
+  private DimensionLayer dimensionLayer;
+  private MapView mapView;
+  private MobileMapPackage mobileMapPackage;
+
+  @Override
+  public void start(Stage stage) {
+    try {
+      // create stack pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene scene = new Scene(stackPane);
+      scene.getStylesheets().add(Objects.requireNonNull(getClass().getResource("/display_dimensions/style.css")).toExternalForm());
+
+      // set title, size, and add scene to stage
+      stage.setTitle("Display Dimensions Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+
+      // set up display label for dimension layer name, and check boxes for controlling visibility and definition expression
+      Label dimensionLayerName = new Label();
+      CheckBox visibilityCheckBox = new CheckBox("Dimension Layer visibility");
+      visibilityCheckBox.setSelected(true);
+      CheckBox defExpressionCheckBox = new CheckBox("Definition Expression:" + "\n" + "Dimensions >= 450m");
+      defExpressionCheckBox.setWrapText(true);
+
+      // add the label and checkboxes to a JavaFX VBox
+      VBox controlsVBox = new VBox(6);
+      controlsVBox.setBackground(new Background(new BackgroundFill(Paint.valueOf("rgba(0,0,0,0.5)"), CornerRadii.EMPTY,
+        Insets.EMPTY)));
+      controlsVBox.setPadding(new Insets(10.0));
+      controlsVBox.setMaxSize(220, 100);
+      controlsVBox.getStyleClass().add("panel-region");
+      controlsVBox.getChildren().addAll(dimensionLayerName, visibilityCheckBox, defExpressionCheckBox);
+      controlsVBox.setDisable(true);
+
+      // create a map view
+      mapView = new MapView();
+
+      // load a mobile map package
+      final String mmpkPath = new File(System.getProperty("data.dir"), "./samples-data/mmpk/EdinburghPylonsDimensions.mmpk").getAbsolutePath();
+      mobileMapPackage = new MobileMapPackage(mmpkPath);
+      mobileMapPackage.loadAsync();
+
+      mobileMapPackage.addDoneLoadingListener(() -> {
+        // check the mmpk has loaded successfully and that it contains a map
+        if (mobileMapPackage.getLoadStatus() == LoadStatus.LOADED && mobileMapPackage.getMaps().size() > 0) {
+          // add the map from the mobile map package to the map view
+          mapView.setMap(mobileMapPackage.getMaps().get(0));
+          mapView.setViewpoint(new Viewpoint(55.908386, -3.305863, 20310));
```

Comment:
```suggestion
          mapView.setViewpoint(new Viewpoint(55.908, -3.305, 20310));
```
Could make the viewpoint less specific